tests/unit/src/main/scala/tests/TestingServer.scala

```diff
@@ -698,6 +698,35 @@ final class TestingServer(
     }
   }
 
+  def indentOnPaste(
+      filename: String,
+      query: String,
+      expected: String,
+      paste: String,
+      root: AbsolutePath,
+      formattingOptions: Option[FormattingOptions]
+  )(implicit loc: munit.Location): Future[Unit] = {
```

Comment:
You can delete brackets which surround `for` and `yield`

Reply:
Why `loc` is needed here?

Reply:
It might not be needed actually, will fix.